app/assets/stylesheets/thing.scss

```diff
@@ -155,6 +155,14 @@ form.simple_form {
       width: 100%;
     }
   }
+  .proquest {
+    label {
+      margin-bottom: 1rem;
+    }
+    .radio.field-subitem {
```

Comment:
Thanks, good catch on that. I'll remove the rule.

Reply:
Is the choice to use a drop-down instead of radio-buttons because the error styling was a mess? Radios seem like a better choice for this to me but we can check in with UX as well to confirm this would be reasonable.

Reply:
Yes. With radio buttons, the preferred structure for a11y purposes is to have a label element for each button, since each button is an input. This causes the validation error message to appear next to the "Yes" radio button, because simple_form renders the message next to the first label it sees.  I'm sure there's a way to hack where the validation message renders, but it seemed preferable to wait for UX input before spending more time on it.

Reply:
I did a bit of poking around this morning, unable to shake this issue from my head. I apologize if this is going to far as a reviewer.

It looks like we're putting a validation message in a different location on the transfer processing form, but I haven't found any other examples of radio buttons that are similar to this (we use radios on thesis processing form, but those are inline).

The radio button gets implemented here: https://github.com/MITLibraries/thing/blob/main/app/views/transfer/_thesis_target.html.erb#L2 , but the validation message appears to show up prior to this line:

https://github.com/MITLibraries/thing/blob/main/app/views/transfer/show.html.erb#L85
(at least when I look at the staging site - transfer ID 50 is one that has sufficient records)

Looking back at the commit, this validation is being handled via jQuery validate, and not whatever built-in process that Rails usually uses? Maybe that's how a custom location is happening? It appears that the placement for this type of validation failure is at https://github.com/MITLibraries/thing/blob/main/app/views/transfer/show.html.erb#L132-L133

Hopefully this helps. I agree with your overall position, though, that getting UX to weigh in on this issue would be helpful before spending too much time on it - so maybe everything I write here should be set aside until then.

Reply:
No, not going too far at all. This turned out to be the most frustrating part of this change, and I'd like to fix it, so I welcome the discussion.

The other difference to note between this situation and the radio buttons in the processing form is that here we are using nested attributes, which requires a slightly different construction. As I understand it, we need to use `fields_for` (or `simple_fields_for`) in order to update the associated model. That may or may not affect it.

All that said, I'd been hoping to avoid jQuery on this, but it may be the best/only way to hack this.

Reply:
@matt-bernhardt I ended up going with the approach you shared. Since we're already doing it elsewhere in the app, it doesn't feel weird to also do it here, and I think it's well worth using radio buttons instead of a select.